CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ if(${tntver} VERSION_LESS 1.7.4.291)
     message(FATAL_ERROR "Tarantool >= 1.7.4-291 is required")
 endif()
 
-if(BUNDLE_RDKAFKA)
+if(STATIC_BUILD)
```

Comment:
ENABLE_BUNDLED_FOO seems to be a standard name for such configuration options.

https://github.com/tarantool/tarantool/blob/d07e3ac2facfc5c5cec43ff138e2bdbd1fab1011/CMakeLists.txt#L304
https://github.com/tarantool/tarantool-c/blob/e9f17d649c2ee543cd505ceeff59e050763528a9/CMakeLists.txt#L38-L39

Reply:
I want a single option that will be the same for all rocks and mean roughly "link all your stuff statically, and don't use anything external". Hence STATIC_BUILD

Reply:
Nothing blocks us with that. We still can add separate options and the helper one. This is the right way, because packaging for distros usually requires linking with system libraries, but sometimes there are problems with some specific system libraries: e.g. lack of them in certain distros.

Hipster way packaging should not break existing agreements. Otherwise it would be the systemd way: a half of the world will hate us.

Reply:
I'm not saying we should remove BUILDLED_*, I'm just saying there should be a universal option that you can use to link all dependencies statically. It could be an alias of BUNDLED_* in case if there is only one dependency. But if there is more than one (which is true in case of kafka), it should statically link everything.

Reply:
As for systemd, there is only a small and vocal hater community, which doesn't even have strong arguments, except the fact that systemd is monolithic.

Reply:
I'm not against. Thanks for the clarification.